### PR TITLE
Update translations

### PR DESCRIPTION
**Pull request by McGiverGim**
Update translations from Crowdin.

Thanks for the patience to all the translators!!!

### CODE REVIEW

**Review by haslinghuis — DISMISSED** [review]

File: locales/ca/messages.json
Comment (benlumley) [inline locales/ca/messages.json:678]: Videos Betaflight 4.3 de Joshua Bardwell ??

**Comment by blckmn** [comment]
AUTOMERGE: (FAIL)
 - github identifies PR as mergeable -> FAIL
 - assigned to a milestone -> PASS
 - cooling off period lapsed -> FAIL
 - commit count less or equal to three -> PASS
 - `Don't merge` label NOT found -> PASS
 - at least one `RN:` label found -> PASS
 - `Tested` label found -> FAIL
 - assigned to an approver -> PASS
 - approver count at least three -> FAIL

File: locales/ca/messages.json
Comment (McGiverGim) [reply locales/ca/messages.json:678]: If you don't like some translation, the only way to fix them is in Crowdin https://crowdin.com/project/betaflight-configurator

**Review by haslinghuis — APPROVED** [review]

**Review by blckmn — APPROVED** [review]

**Pull request merged**
